js/PulsarUIComponent.js

```diff
@@ -158,22 +159,62 @@ PulsarUIComponent.prototype.initDataTables = function () {
         $($.fn.dataTable.tables(true)).DataTable().columns.adjust().responsive.recalc();
     });
 
+    // Dropdowns within responsive tables will be hidden by overflows, this method detaches the dropdown menu and reattaches it on the body
+    $(document).on('show.bs.dropdown', '.datatable', function(e) {
+        var dropdown = $(e.target),
+            dropdownMenu = dropdown.find('.dropdown__menu'),
+            dropdownMenuTop = dropdown.offset().top,
+            dropdownMenuLeft = dropdown.offset().left;
+
+        dropdownMenu.css({
+            display: 'block',
+            left: dropdownMenuLeft - dropdownMenu.width() + dropdown.width() - 2,
+            top: dropdownMenuTop + dropdown.height() - 1
+        }).addClass('js-row-actions-dropdown');
```

Comment:
Same with `width()` here, although these are admittedly micro optimisations I think we should still strive to avoid reflows where we can in a readable friendly fashion. We could perhaps get the width and store it once.